pkg/resources/pciDevice.go

```diff
@@ -70,8 +71,11 @@ func NewPciDevice(dev *ghw.PCIDevice, rFactory types.ResourceFactory, infoProvid
 	if len(infoProviders) == 0 {
 		infoProviders = append(infoProviders, rFactory.GetDefaultInfoProvider(pciAddr, driverName))
 	}
+	nodeNum := -1
+	if !rc.ExcludeTopology {
```

Comment:
It changes current defaults. We shouldn't do it or at least add a upgrade note

Reply:
The default is to advertise numa topology and that is maintained.
The default value of ExcludeTopology is false.